notebooks/dynamicrafter-animating-images/dynamicrafter-animating-images.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    dynamicrafter_path.mkdir(parents=True, exist_ok=True)
</pre><p>this part looks strange, because you create directory and then check that it does not exists, repo clonning never happens</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Moved</p>